src/core/namespace.c

```diff
@@ -97,6 +97,23 @@ static const TargetMount protect_kernel_tunables_table[] = {
         { "/sys/fs/cgroup",             READWRITE,      false, }, /* READONLY is set by ProtectControlGroups= option */
 };
 
+/*
+ * ProtectHome=read-only table, protect $HOME and $XDG_RUNTIME_DIR and rest of
+ * system should be protected by ProtectSystem=
+ */
+static const TargetMount protect_home_read_only_table[] = {
+        { "/home",                      READONLY,       true,   },
+        { "/run/user",                  READONLY,       true,   },
+        { "/root",                      READONLY,       true,   },
+};
+
+/* ProtectHome=true table */
```

Comment:
"ProtectHome=yes", otherwise one has to think for a second if this is a copy*paste error or not.